rest/route/patch_test.go

```diff
@@ -829,9 +830,17 @@ buildvariants:
 		Id:                   mgobson.NewObjectId(),
 		Project:              projectRef.Id,
 		Githash:              "3c7bfeb82d492dc453e7431be664539c35b5db4b",
-		PatchedParserProject: config,
+		ProjectStorageMethod: evergreen.ProjectStorageMethodDB,
+		PatchedProjectConfig: config,
 	}
 	require.NoError(t, unfinalized.Insert())
+
+	pp := &serviceModel.ParserProject{}
+	err := util.UnmarshalYAMLWithFallback([]byte(config), &pp)
```

Comment:
Go is smart enough to make it work anyways, but it's not necessary to take the pointer of a pointer here. Just passing in `&serviceModel.ParserProject{}` is sufficient.

Same with the test below.

Reply:
oops. Removed the `&`.